prow/cmd/status-reconciler/main.go

```diff
@@ -23,10 +23,11 @@ import (
 
 	"github.com/sirupsen/logrus"
 	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/test-infra/prow/config"
```

Comment:
super nit: this import (and the one below) should be grouped with the rest of the test-infra ones

Reply:
Thank you for the review. Done.